Cesium3DTiles/src/Tile.cpp

```diff
@@ -238,6 +238,7 @@ void Tile::loadContent() {
       .thenInWorkerThread(
           [loadInput = std::move(loadInput),
            projections = std::move(projections),
+           ownerTileset = &tileset,
```

Comment:
it maybe dangerous to pass raw pointer to the lambda that will be passed to another thread since tileset can be destroyed before the lambda get called in the main thread (But that is a bug if it really does currently). I think it's better to capture the value of `getGltfUpAxis()` in the lambda so we don't have to manage object lifetime. Something like
```
uint8_t axisUp = tileset.getGltfUpAxis();
std::move(maybeRequestFuture.value()).thenInWorkerThread([axisUp]() { /*do something with it*/});
```

Reply:
Indeed that's much better. (I usually try to "pass on as little information as possible", but am not sure why I didn't do this here)